.devcontainer/docker-compose.yml

```diff
@@ -23,7 +23,11 @@ services:
     environment:
       PGADMIN_DEFAULT_EMAIL: ${PGADMIN_DEFAULT_EMAIL:-pgadmin4@pgadmin.org}
       PGADMIN_DEFAULT_PASSWORD: ${PGADMIN_DEFAULT_PASSWORD:-admin}
+      PGADMIN_LISTEN_PORT: 5050
```

Comment:
This is not mandatory as we can forward the internal pgadmin port (80 by default) to any external port

Reply:
It turns out that it really wasn't needed.